Grimoire-shared/src/main/java/io/github/crucible/omniconfig/api/properties/IAbstractProperty.java

```diff
@@ -14,31 +14,31 @@
  * @author Aizistral
  */
 
-public abstract interface IAbstractProperty {
```

Comment:
Also this. I know all interfaces are inherently abstract, but I mark them as such for the purpose of indication that nothing implements this interface on its own, only interfaces that extend this one.